tasks/task_14/server/service/service.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import (
+	"strings"
+)
+
+type Service struct {
+}
+
+func NewService() *Service {
+	return &Service{}
+}
+
+func (s *Service) CheckString(str string) (string, error) {
```

Comment:
this is not service, this is validator 

and this method should be `Validate(ctx context.Context, str string) error`